docs/create-webhook.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       description: The github repo name
       default: "repo"
     - name: GithubSecretName
-      description: The secret name for github
+      description: The secret name for github access token
```

Comment:
access token, and optionally the webook secret

Reply:
I removed these non-getting-started changes from this PR, ty!